packages/migrate/src/utils/ensureDatabaseExists.ts

```diff
@@ -210,7 +210,9 @@ export function getDbLocation(credentials: DatabaseCredentials): string | undefi
     return credentials.uri!
   }
 
-  if (credentials.host && credentials.port) {
+  if (credentials.socket) {
```

Comment:
This should also include the `host` query parameter for PostgreSQL: https://www.prisma.io/docs/orm/overview/databases/postgresql#connecting-via-sockets Not sure how/if this is accessible here as `credentials.host` seems to be the parsed hostname though.

Reply:
I'll handle the socket path supplied as host for PostgreSQL here. I think given that it's a unix socket I'll look for a host value that appears to be a unix-style path pattern.

Reply:
All set